slime.el

```diff
@@ -6518,7 +6518,8 @@ If ARG is negative, move forwards."
   (slime-eval-describe `(swank:pprint-inspector-part ,part)))
 
 (defun slime-inspector-eval (string)
-  "Eval an expression in the context of the inspected object."
+  "Eval an expression in the context of the inspected object.
+The inspected object will be bound to the \"*\" variable."
```

Comment:
It's variables that are bound to objects, isn't it? Or does the nomenclature work both ways? :-)

Reply:
Good point! You'll see I caught this in the manual, but not in the docstring.  I will fix and re-push.